docs/notebooks/JAX_for_the_impatient.ipynb

```diff
@@ -0,0 +1,1293 @@
+{
```

Comment:
<p>I don't think the note at the end is necessary -- the expectation in this case /is/ the average as you wrote above.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/flax/pull/542/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>

Reply:
<p>Well it depends on how you cast the problem, the expectation of the mean here is the "true" expectation. However, the other way is not true unless you consider that your sample fits the actual data distribution. The reason I made the note is that usually you start from a probability distribution (from which you sample come from), and you cast the optimization problem on the expectation, then solve it for the sampled one (that we have here). </p><p> </p><p>The reason I made the note is that I don't want people to nitpick on exactly that, since depending on how you usually cast the problem you might disagree with this one here. I'll rephrase the note a bit though.</p>